docs/spark-writes.md

```diff
@@ -302,6 +302,14 @@ data.writeTo("prod.db.table")
     .createOrReplace()
 ```
 
+You can specify an Iceberg table location such as the `LOCATION` clause in SQL by add the `location` paramter to the `tableProperty`:
```

Comment:
Nit: I know there's a bad example above, but typically we avoid 'you' or 'me' in documentation.  you can see other reviews, for example:  https://github.com/apache/iceberg/pull/4301#discussion_r823197799

Also, I'm not sure why we reference "LOCATION" clause in SQL, which is documented elsewhere.

How about:
```The Iceberg table location can also be specified by the `location` table configuration```

Let me know what you think

Reply:
Thanks for reviewing this and the great suggenstion, Szehon! The following your suggestion is simpler and makes sense. Based on your suggestion, I updated a bit; `... table configuration` to `table property`. How about this?

`The Iceberg table location can also be specified by the `location` table property.`


For your comment;

> Nit: I know there's a bad example above, but typically we avoid 'you' or 'me' in documentation. you can see other reviews, for example: https://github.com/apache/iceberg/pull/4301#discussion_r823197799

Thanks for pointing out, I understand it.

> Also, I'm not sure why we reference "LOCATION" clause in SQL, which is documented elsewhere.

I tried saying the `tableProperty("loaction", "...")` is the same as `LOCATION` in SQL, but the suggestion looks better.

Reply:
Yes that will work.  Thanks!